conda-forge.yml

```diff
@@ -2,7 +2,7 @@ github:
   user_or_org: nsls-ii-forge
 channels:
   sources:
-    - lightsource2-tag
+    - nsls2forge
     - defaults
```

Comment:
Should this include conda-forge as well?

Reply:
Not now, sorry. We are trying to produce a set of packages using `defaults` and `nsls2forge` for now.